### ISSUE TITLE
Fields do not obey validation rules

### ISSUE THREAD

**Issue by matthijssm** [outl1ne/nova-page-manager#126]
When I use rules like this `->rules("required", "string", "min:3")` nothing really happens when I give wrong input. I would expect an error message.

**Comment by cliche23** [comment]
Yes, we've run in the same issue. Any projections when this could be fixed?

**Comment by vladimirovDi** [comment]
I have same issue, when it will be fixed?

**Comment by cliche23** [comment]
Any update on this?

**Comment by t1mofe1** [comment]
@KasparRosin @Tarpsvo Hey, issue seems to be open already for 1.5 years.. Have any updates on this?
